app/controllers/users_controller.rb

```diff
@@ -0,0 +1,4 @@
+class UsersController < ApplicationController
+  def profile
```

Comment:
I know its probably not a big issue, but is it problem to change the 'show' method into a 'profile' method?